WalletWasabi.Backend/wwwroot/about.html

```diff
@@ -0,0 +1,281 @@
+                        </li>
+                        <li class="nav-item">
+                            <a class="btn btn-secondary" href="index.html#download">Download</a>
+                        </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+    <div class="fixed-img">
+        <img src="img/05_wallet.png" alt="Wasabi Wallet - About">
+        <div class="floating-title">Better money.
+            <br>Better world.</div>
+    </div>
+
+    <div class="content">
+        <section class="blured-section p-100">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-8">
+                        <h1 class="title-lined left">Motivation</h1>
+                        <p>
+                            We believe that the most urgent problem for mankind is fixing the money, that Bitcoin is the best option for doing this and that privacy is its biggest problem.
+                        </p>
+                        <p>
+                            Wasabi Wallet was created to help Bitcoin maintain its cypherpunk roots on its path to a global reserve currency and to mitigate the inherent privacy issues of its public ledger. This is why Wasabi Wallet is an open-source, non-custodial, privacy-focused Bitcoin wallet for desktop that implements trustless coinjoin magic.
+                        </p>
+                        <p>
+                            After years of doing privacy research and development, we’ve concluded that having good tools is not enough - They need to be easy to use, otherwise they will only be utilized by the minority, making users stand out from the crowd. To increase the use of these tools, Wasabi is designed to be user friendly and to provide privacy automatically by default.
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="subpage p-100">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-12">
+                        <h2 class="title text-center">The history of Wasabi Wallet</h2>
+                        <ul class="timeline">
+                            <li>
+                                <div class="timeline-title">2015 - YEARS OF RESEARCH</div>
+                                <p>Ádám Ficsór, aka nopara73, and Lucas Ontivero conducted extensive research on Joinmarket, Zerolink, Tor and Bitcoin Programming in their quest to create a privacy-oriented bitcoin wallet of their own.</p>
+                            </li>
+                            <li>
+                                <div class="timeline-title">2018 - zkSNACKs Ltd. IS FORMED</div>
+                                <p>Nopara73 collaborated with Gergely Hajdú and Bálint Harmat to create zkSNACKs, a for-profit organization that sponsors the development of Wasabi Wallet, previously called Hidden Wallet.</p>
```

Comment:
in between should be a mentino of hidden wallet.

Reply:
I added the place, pls write the text.